cpp/src/aztec/benchmark/plonk_bench/plonk.bench.cpp

```diff
@@ -37,6 +37,9 @@ void construct_witnesses_bench(State& state) noexcept
     for (auto _ : state) {
         waffle::StandardComposer composer = waffle::StandardComposer(static_cast<size_t>(state.range(0)));
         generate_test_plonk_circuit(composer, static_cast<size_t>(state.range(0)));
+        state.PauseTiming();
```

Comment:
You are also measuring circuit construction, so it doesn't just show witness construction time

Reply:
Yeah I agree - I was trying to preserve the original state of the test (just had to add compute_proving_key to fix a segfault) but I went ahead and moved the timing as you suggested. I'll do a follow on to fix the rest of the plonk bench since they still don't run completely.